templates/components/select.build.hbs

```diff
@@ -1,26 +1,11 @@
 <div v-if="description" class="help-block description">\{{ description }}</div>
-<template v-if="readonly">
-  <p class="form-control-static">\{{ value }}</p>
-    <select
-      class="form-control hidden-select hidden focus-outline"
-      :name="name"
-      :id="name"
-      v-model="value"
-      v-on:change="updateValue()"
-      v-on:input="onInput($event)"
-      tabindex="0"
-    >
-      <option v-if="placeholder" value="">\{{ placeholder }}</option>
-      <option v-for="option in options" :value="(_.isNumber(option.id) || _.isString(option.id)) ? option.id : option.label" :disabled="option.disabled">
-        \{{ option.label || option.id }}
-      </option>
-    </select>
-</template>
-<template v-else>
+
```

Comment:
Can you please remove new empty blank line

Reply:
done changes